### PR TITLE
`Stringifiers` for all the IDs when converting to String

### PR DESCRIPTION
**Pull request by dmdashenkov**
Before we had `Stringifiers`, `Identifiers` and `IdTransformer` for converting the IDs to `String` and vise versa. Now it's only the duty of `Stringifiers`.
After migrating to core v0.8.31 ([#419](https://github.com/SpineEventEngine/core-java/pull/419)) the `IdTransformer` class should be removed at all.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: gcd/src/main/java/org/spine3/server/storage/datastore/DsRecordStorage.java [hunk 1]
```diff
@@ -137,11 +141,11 @@ public boolean delete(I id) {
         }
 
         final EntityRecord record = Entities.entityToMessage(response, RECORD_TYPE_URL);
-        final LifecycleFlags entityStatus = getEntityStatus(response);
-        final EntityRecord result = isDefault(entityStatus) // Avoid inequality of written and read records
-                                    ? record                // caused by empty {@code EntityStatus} object
+        final LifecycleFlags flags = getLisececleFlags(response);
```
Comment (armiol) [inline gcd/src/main/java/org/spine3/server/storage/datastore/DsRecordStorage.java]: Please, do not create any commits in a hurry.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov see my comment.

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

I suppose this to be merged into `master` after the `entity-storage-fields` branch is merged.

**Pull request closed without merging**